### ISSUE TITLE
Issue with resolving .bootstraprc file path

### ISSUE THREAD

**Issue by Animagani** [shakacode/bootstrap-loader#75]
Hello,

I have an issue connected with resolving path to .bootstraprc file. 

**Background:**
On CI server, before running build scripts I am symlinking node_modules from different directory. It let's me to save build time, as I don't have to install node_modules on each build (gitlab-ci is odd and currently it's not possible to save node_modules directory between builds).

**Issue:**
On build, bootstrap-loader tries to find .bootstraprc file in path /foo/bar/cache/.bootstraprc (node_modules are stored in /foo/bar/cache/node_modules, project is cloned to /foo/bar/builds/projectX and build is running there).

**Possible solution:**
I've changed __bootstrap-loader/src/bootstrap.config.js__ file:

```javascript
// before
const userConfigPath = path.resolve(__dirname, `../../../${CONFIG_FILE}`);
// to
const userConfigPath = path.join(process.cwd(), CONFIG_FILE);
```
But I'm not sure if it doesn't break something else ;)

Other node-modules with their configurations in project root directory (eslint, babel) works correctly.

**Comment by damianlegawiec** [comment]
Same here when running in Docker container. We made some workarounds for this but a proper resolution would be great.

**Comment by justin808** [comment]
I'd love to take a well tested and document PR for this, if you have one!

**Comment by phryneas** [comment]
same problem here with a symlinked node_modules folder.
The solution from @Animagani works well, it would be great, if you could intergrate it.

**Comment by vjpr** [comment]
@phryneas Same...had a symlinked `node_modules` which caused the issue. Great to find a simple fix.

**Comment by justin808** [comment]
@vjpr, @phryneas, @damianlegawiec, @Animagani Should we have a separate PR? or put this into #90?

CC: @pherris

All, really appreciate the help. I'm jamming on some other projects at https://github.com/shakacode, most notably those with react-native along with [React on Rails](https://github.com/shakacode/react_on_rails/] and the [associated tutorial]( https://github.com/shakacode/react-webpack-rails-tutorial/). @alexfedoseev is super busy as well!

**Comment by justin808** [comment]
@vjpr, @phryneas, @damianlegawiec, @Animagani:

See #114. I've published 1.1.0-beta.1. I'm awaiting review comments and somebody to verify that the new build works.
